go/kbnm/install_host

```diff
@@ -16,29 +16,54 @@ realpath() {
   fi
 }
 
```

Comment:
This installer and related templates are obsolete now. Worth keeping or nuke it?